en/micro-integrator/docs/references/connectors/mongodb-connector/mongodb-connector-example.md

```diff
@@ -0,0 +1,257 @@
+# MongoDB Connector Example
+
+The MongoDB Connector can be used to perform CRUD operations in the local database as well as in MongoDB Atlas (cloud version of MongoDB).
+
+## What you&#39;ll build
+
+This example explains how to use MongoDB Connector to insert and find documents from MongoDB database.
+
+The sample API given below demonstrates how the MongoDB connector can be used to connect to the MongoDB Server and perform **insert many and find** operations on it.
+
+- `/insertmany`: The user sends the request payload which includes the connection information, collection name and the documents to be inserted. This request is sent to WSO2 EI by invoking the MongodbConnector API. It will insert the documents into the MongoDB database.
+
+    <p><img src="../../../../assets/img/connectors/mongodb-conn-1.png" title="Insert many function" width="800" alt="Insert many function" /></p>
+
+- `/find`: The user sends the request payload, containing the connection information, collection name and the find query. This request is sent to WSO2 EI by invoking the MongodbConnector API. Once the API is invoked, it returns the documents matching the find query.
+
+    <img src="../../../../assets/img/connectors/mongodb-conn-2.png" title="Find function" width="800" alt="Find function"/>
+
+If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
+
+## Connect to MongoDB Atlas
```

Comment:
@shiromi-basil 
Where do we do these steps?
Seems like a 'before you begin' or 'prerequisites'

If this is a prerequisite for 'Configuring the Connector in the Studio', we can add the instructions there.
Could you check?

Reply:
Resolved in the commit https://github.com/shiromi-basil/docs-ei/commit/e5050db516842231ba0c3e6e70a951d5bef6f9e6.